components/prism-hoon.js

```diff
@@ -0,0 +1,30 @@
+Prism.languages.hoon = {
+	constant: /(?:%\.n)|(?:%\.y)|(?:%[\w-]+\w?)/,
+	comment: {
```

Comment:
Please use quoted keys for all token names. So `constant` and `comment` have to be changed.

```suggestion
	'comment': {
```